docs/services/allied_vision_camera.rst

```diff
@@ -3,6 +3,8 @@ Allied Vision Camera
 
 This service controls an Allied Vision camera. It is a wrapper around the Vimba SDK, which requires its installation.
 The service uses the Python API for Vimba SDK, called ``VimbaPython``.
+In order to be able to run the service on an Allied Vision Camera, the right Driver needs to be set up on the device.
+The Allied Vision Camera's driver can be changed through NI MAX.
```

Comment:
Let's be more specific. You can simplify the first sentence to:
"To be able to run this service, the camera needs to be set up with the right Allied Vision USB driver."
And then we would need more information included in the second sentence:
"If the device is also controlled through other interfaces, for example NI MAX, you need to switch back to the right driver through that interface."

Don't hesitate to change some of my wording if you have other ideas.